test/optics/interference_films_from_angle_refraction_refractive_index_wavelength_test.py

```diff
@@ -0,0 +1,56 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from sympy import pi
+from symplyphysics import (units, SI, convert_to, Quantity, errors)
+from symplyphysics.laws.optics import interference_films_from_angle_refraction_refractive_index_wavelength as thickness_law
+
+# Description
+## Let the wavelength be 1500 nanometer, the order of interference is 1, the angle of refraction is
+## 60 degree (pi / 3 radian), and the refractive index is 5. Then the film thickness will be 300 nanometer.
+## https://www.indigomath.ru//raschety/uHv0Ia.html
```

Comment:
Use some external example. This synthetic example does not verify your understanding of the law.

Reply:
Well you've found somewhat valid example but numbers do not match.